packages/lists/src/StaticList/StaticList.Example.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { StaticList } from './StaticList';
+
+const ITEMS = new Array(25).fill(0);
+let APPEND_EXAMPLE_ITEMS = [...ITEMS];
+
+function appendItems(): void {
+  APPEND_EXAMPLE_ITEMS = [...APPEND_EXAMPLE_ITEMS, ...new Array(10).fill(0)];
+}
+
+class PerfMark extends React.PureComponent<{ children?: React.ReactNode }> {
+  private _start: number;
+  private _type: Function | string;
+  private _key: string;
+
+  constructor(props: { children?: React.ReactNode }) {
+    super(props);
+
+    this._start = performance.now();
+    this._key = (this.props.children as { key: string }).key;
+    /* tslint:disable-next-line:no-any */
+    this._type = this._getComponentTypeAsString((this.props.children as any).type);
+  }
+
+  public componentDidMount(): void {
+    const end: number = performance.now();
+    const delta: number = end - this._start;
```

Comment:
**TODO:** Looks like this is reporting total mount time not individual component render time, need to dig.